### ISSUE TITLE
forge.message.listen's reply message is picked up by wrong activation

### ISSUE THREAD

**Issue by jaredsohn** [trigger-corp/browser-extensions#35]
Here is a simplified version of my extension that exhibits the problem:
1. Background page has:

forge.message.listen(
  function(scriptname, reply) {
     reply(true);
  }, function(err) {
      console.log(err);
  }
);
1. I have at least two activations that consist of the following code:

var __activation_id = "$SCRIPT_NAME"; // set via tokenizing code
forge.message.broadcastBackground(__activation_id, __activation_id, 
  function(content) {
    console.log(__activation_id + " response: " + content)
}, function(content) {
    console.log(__activation_id + " response: " + content)
});

Basically when I run this, my activations all talk to the background page and I would expect each to write to the console the reply they received.  Unfortunately, I often experience one activation getting multiple responses and the others not getting any responses.  There isn't a parameter to restrict the reply to certain listeners, but I would think that could and would be determined automatically.
